functions/menu.bash

```diff
@@ -138,8 +138,10 @@ show_main_menu() {
     "43 | Reverse Proxy"     "Setup Nginx with password authentication and/or HTTPS access" \
     "44 | Delay rules load"  "Delay loading rules to speed up overall startup" \
     "   | Default order"     "Reset config load order to default (random)" \
+    "45 | Use AdoptOpenJDK"  "Install AdoptOpenJDK as primary java provider" \
```

Comment:
would need this to be a list of available java choices to select from (like 44 above and the line below)

Reply:
Then we should probably convert the 63 option to be configured here as well then, right?

Reply:
yes

Reply:
Fixed